recipes/abyss/meta.yaml

```diff
@@ -9,7 +9,7 @@ build:
   number: 0
 
 source:
-  url: http://www.bcgsc.ca/platform/bioinfo/software/{{ name|lower }}/releases/{{ version }}/{{ name|lower }}-{{ version }}.tar.gz
```

Comment:
@sjackman if you do not increase the build number (or the version number and reset the build number to 0) the pkg will not be build.

Reply:
Abyss version 2.1 were never previously built due to blacklisted so I think this is good to go.